01-tree-specification.bs

```diff
@@ -218,28 +218,26 @@ The resulting values of the evaluation of the `tree:path`, are the values that m
 When multiple results from the path are found, they need to be interpreted as a logical OR: at least one of these values will fulfill the comparator.
 
 A client, in case it wants to process relations that use the `tree:path` property, MUST implement a matching algorithm to check whether the relation is relevant.
-I.e., a `tree:path` on `(rdfs:label [sh:alternativePath rdfs:comment ] )` will be useful when the client is tasked to filter on `rdfs:comment`.
+E.g., a `tree:path` on `(rdfs:label [sh:alternativePath rdfs:comment ] )` will be useful when the client is tasked to filter on `rdfs:comment`.
 
 Note: A server is allowed to refer the `tree:path` to a property that is not materialized in the current response. For the client, if it also needs those triples, we assume in this spec that the client has another way of retrieving those, or already retrieved them from another source.
 
-## Comparing strings, named node and time literals ## {#sparqlComparators}
+## Comparing strings, named nodes and time literals ## {#sparqlComparators}
 
 When using comparator relations such as `tree:GreaterThanRelation` and `tree:PrefixRelation` the values MUST be compared as defined in the [SPARQL algebra functions](https://www.w3.org/TR/sparql11-query/#expressions).
-See the [`tree:Relation`](#Relation) section.
-See the sections below for recommendations and extensions to SPARQL behavior.  
+See the [`tree:Relation`](#Relation) definitions in the vocabulary for the specific references to those functions.
 
-### Recommendation for Time Literal Definition ### {#time}  
+Following the definitions of the SPARQL functions, **string comparisons** MUST be performed using unicode canonical equivalence, and ordering MUST be done using case sensitive unicode ordering.
```

Comment:
SPARQL really specify that? Looking at [this section](https://www.w3.org/TR/sparql11-query/#OperatorMapping) it does not seems so, but maybe I am wrong. But, if it is specified why do we add it into the spec since we say to "simply" SPARQL.

Reply:
Oh, I form reading the next sentence, maybe it mean that we have to follow SPARQL while considering the addition requirement written here?

Reply:
I’m actually quite confused at this moment: I remembered it did do that already, and I wanted to add this text to be extremely clear about it again in the TREE spec, but I cannot find it again in the SPARQL or XPath functions specs. To be further investigated